lib/phoenix/router.ex

```diff
@@ -358,9 +358,11 @@ defmodule Phoenix.Router do
             conn
         rescue
           e in Plug.Conn.WrapperError ->
+            :telemetry.execute([:phoenix, :router_dispatch, :failure], %{}, %{error: e})
```

Comment:
According to [this issue](https://github.com/beam-telemetry/telemetry/issues/57), it should probably be something like this:

```suggestion
            duration = System.monotonic_time() - start
            :telemetry.execute([:phoenix, :router_dispatch, :failure], %{}, %{kind: :error, reason: e, stacktrace: __STACKTRACE__})
```

Can you please update both accordingly?